app-backend/ingest/src/main/scala/com/azavea/rf/ingest/Ingest.scala

```diff
@@ -143,15 +150,25 @@ object Ingest extends SparkJob {
       }
 
     val layerRdd = ContextRDD(multibandTiledRdd, tileLayerMetadata)
-    val (writer, attributeStore) = getRfLayerWriter(layer.output)
+    val (writer, deleter, attributeStore) = getRfLayerManagement(layer.output)
     val sharedId = LayerId(layer.id.toString, 0)
 
+    if (overwriteLayer) {
+      try {
+        deleter.delete(sharedId)
+        attributeStore.delete(sharedId)
+      } catch {
+        case e: Throwable => throw e
```

Comment:
This will be an area where some cleanup is going to be necessary. As of now, it is just a placeholder.